waspc/data/Generator/templates/server/src/dbClient.js

```diff
@@ -12,6 +13,41 @@ const createDbClient = () => {
   const prismaClient = new Prisma.PrismaClient()
 
   {=# isAuthEnabled =}
+
+  prismaClient.$use(async (params, next) => {
+    // Ensure strong plaintext password. Must come before hashing middleware.
+    // Throws an EntityValidationError on the first validation that fails.
+    if (params.model === '{= userEntityUpper =}' && params.action === 'create') {
+        const data = params.args.data || {}
+
+        const defaultValidations = [
+          { name: 'email must be present', fn: data => !!data.email },
```

Comment:
`fn` here stopped me for just a moment, as I was considering if there is a better name. Maybe `validator`? Or `validationFn`? But on the other hand, context here (which is "validation") should always be very obvious I think, so it can be ok to have such short names. Ok, I think this is ok, just thinking out loud!

Reply:
I was going back and forth too on `fn` vs `validator`. 😄 I think since it caught your eye too maybe it is a sign we should go with `validator` perhaps just to cement it?

Reply:
Sounds good!

Reply:
This has been addressed here: https://github.com/wasp-lang/wasp/pull/376/commits/3ffbc62c75cf4a43363b90fc8c2b1480309891ba Thanks!